src/en/vyvymanga/build.gradle

```diff
@@ -1,8 +1,9 @@
 ext {
     extName = 'VyvyManga'
     extClass = '.VyvyManga'
-    extVersionCode = 5
-    isNsfw = true
+    themePkg = 'madara'
+    baseUrl = 'https://vyvymanga.org'
+    overrideVersionCode = 0
 }
```

Comment:
It's still NSFW, check: all manga -> most views. So much yaoi on the first page.

Reply:
> It's still NSFW, check: all manga -> most views. So much yaoi on the first page.

Cool, I saw it, but I forgot to add it